### PR TITLE
Add input clear button background instruction

### PR DESCRIPTION
**Pull request by TheZoker**
Add instructions on how to customize the background color of the input clear button

### CODE REVIEW

File: src/pages/docs/controls.mdx [hunk 1]
```diff
@@ -145,3 +145,22 @@ By default the first primitive (string, number, integer) element will be used.
...
+    jsonforms: { input: { delete: { background: '#f44336' }}}
+  }}
+>
+  <JsonForms
+    ...
+  />
+</ThemeProvider>
+```
+
+If you want to customize the background of the input clear button (the one you can see, if you hover of the input field), you can need to wrap you `JsonForms` component in an `ThemeProvider` component and provide your desired background color to the theme.
```
Comment (sdirix) [inline src/pages/docs/controls.mdx]:
```suggestion
The background of the the 'clear input button' (the one you can see when you hover of the input field) is by default your theme's `palette.background.default` color. If you want to customize the background of this button you can use the custom theme variable `jsonforms.input.delete.background`.
```

File: src/pages/docs/controls.mdx [hunk 2]
```diff
@@ -145,3 +145,22 @@ By default the first primitive (string, number, integer) element will be used.
 In the following example each element will be labeled with its name instead of its message because we provide the name prop.
 
 <ExampleWithArrayLabel />
+
+
```
Comment (sdirix) [inline src/pages/docs/controls.mdx:162]:
```suggestion
## Theming
```

File: src/pages/docs/controls.mdx [hunk 3]
```diff
@@ -145,3 +145,22 @@ By default the first primitive (string, number, integer) element will be used.
...
+### Use custom theme to change input clear background
```
Comment (sdirix) [inline src/pages/docs/controls.mdx]:
```suggestion
### Customize 'clear input button' background
```

**Review by sdirix — CHANGES_REQUESTED** [review]
Please update the PR. Can you post a screenshot of how the background color looks like with the latest JSON Forms?

**Comment by TheZoker** [comment]
This is what the button looks within the docs:
image

And this is the default:
image

**Pull request merged**